src/server/src/database/databaseClient.js

```diff
@@ -0,0 +1,44 @@
+export class DatabaseClient {
+  #client;
+  
+  /**
+   * A function to instantiate the MongoClient object and populate connection fields
+   */
+  #init() {
+    if (this.#client == null) {
+      const MongoClient = require('mongodb').MongoClient;
+      const { mongoUri } = require('../config');
+      this.#client = new MongoClient(mongoUri, { useNewUrlParser: true, useUnifiedTopology: true });
+    }
+  }
+
+  /**
+   * A function to get all objects stores in `maps` collection
+   * @returns a promise for an array of the objects in the `maps` collection 
+   */
+  async getAllMaps() {
+    if (this.#client == null) {
+      this.#init();
+    }
+    
+    try {
+      // Connect the client to the server
```

Comment:
Is this something that we can do once and leave on for other requests? Or do we run this every time... If the latter it might be better in a separate function for reuse